vars/govuk.groovy

```diff
@@ -39,6 +39,17 @@ def buildProject(Map options = [:]) {
       defaultValue: false,
       description: 'Identifies whether this build is being triggered to test a change to the content schemas'
     ),
+    //not for use before content schemas have been merged into publishing api
+    booleanParam(
+      name: 'USE_PUBLISHING_API_FOR_SCHEMAS',
+      defaultValue: false,
+      description: 'DO NOT USE YET: Identifies whether to use publishing api or content schemas for schemas - flag to enable switchover'
+    ),
+    stringParam(
+      name: 'PUBLISHING_API_BRANCH',
```

Comment:
```suggestion
      name: 'PUBLISHING_API_SCHEMAS_BRANCH',
```

I feel like it would be better to have this more specific to indicate it's purpose is for schemas.

It'll also mean it avoids clashing with an existing use of PUBLISHING_API_BRANCH:

https://github.com/alphagov/gds-api-adapters/blob/2f003aa7a5b808e4bd7d57224545abd6fc606af8/Jenkinsfile#L12-L17

Reply:
Now removed this is favour of using the existing SCHEMA_BRANCH variable